victron_ble/cli.py

```diff
@@ -20,6 +20,12 @@ def convert(self, value, param, ctx):
                 addr = parts[0].strip()
                 key = parts[1].strip()
                 return (addr, key)
+            if len(parts) > 2:
+                parts = value.split("|")
```

Comment:
I'm not a fan of this since | has special meaning in bash and will likely lead to lots of confusion if the entire string is not properly quoted.